src/components/Googlebooksapi/Googlebooksapi.tsx

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+const apiKey = 'AIzaSyBCCc8FkvKZmz-Hzw0m3iJbkynPczXub1U';
```

Comment:
put api keys in a .env file please. there should be a guide on using .env variables in React

Reply:
This part will not be needed the correction is done in the feature-googleapi-ui branch

Reply:
api keys are not safe to be committed. Remove the API key before committing in this file as well since the work was done in both branches.
```suggestion
const apiKey = 'test-api-key';
```